README.md

```diff
@@ -410,6 +410,27 @@ For example this means that by default we upgrade the minimum version of Airflow
 to 2.3.0 in the first Provider's release after 11th of October 2022 (11th of October 2021 is the date when the
 first `PATCHLEVEL` of 2.2 (2.2.0) has been released.
 
+Since providers are often connected with some stakeholders that are vitally interested in maintaining
+their integrations (for example cloud providers, ore specific service providers), and we are also bound with
+the Apache Software Foundation release policies. The provider's governance model is something we name
+"mixed governance".
```

Comment:
- Please clarify "we are also bound with the Apache Software Foundation release policies"
- "providers are often connected with some stakeholders" are those stakeholders documented somewhere?
- Please clarify "The provider's governance model"
- You throw in the term "mixed governance" but don't refer to it anywhere. Can you elaborate on it?

Reply:
> * Please clarify "we are also bound with the Apache Software Foundation release policies"

Added link and describe that it's about "who and how releases the ASF software"

> * "providers are often connected with some stakeholders" are those stakeholders documented somewhere?

No, but you do not have to be a stakeholder to propose to release an older version of a provider if you wish to take the burden of cherry-picking and testing the changes. So I do not think we need to document those. Just a PR to an old branch and confirmation that it's been tested is enough.

> * Please clarify "The provider's governance model"

Did.

> * You throw in the term "mixed governance" but don't refer to it anywhere. Can you elaborate on it?

Did.